src/kvstore/raftex/RaftPart.cpp

```diff
@@ -594,7 +596,8 @@ void RaftPart::replicateLogs(folly::EventBase* eb,
                    prevLogTerm,
                    pHosts = std::move(hosts)] (folly::Try<AppendLogResponses>&& result) mutable {
             VLOG(2) << self->idStr_ << "Received enough response";
-            CHECK(!result.hasException());
+            // TODO: since we kill some peers in ut, this may fail, handle exception if necessary
+            // CHECK(!result.hasException());
```

Comment:
I mock disconnect from one Host, and remove it from its peer, and it may throw exception like `runtime_error: Not enough futures` because of the Host has been destroyed.

Reply:
if you want to mock host disconnect,  you should stop the host directly instead of removing it from others's peers.

Reply:
Yeah, you're right. It only throw the exception when I modify the peers. But do you think it is necessary to mock network partition?

Reply:
Of course, we will mock the network partition situation.  We could leave the job in our raft test framework.